definitions/npm/react-splitter-layout_v3.x.x/flow_v0.53.x-/react-splitter-layout_v3.x.x.js

```diff
@@ -0,0 +1,25 @@
+declare type $$reactsplitterlayout$$ReactElement =
+  | boolean
+  | string
+  | number
+  | React$Element<any>;
+
+declare type $$reactsplitterlayout$$Props = {|
+  +customClassName?: string,
+  +vertical?: boolean,
+  +percentage?: boolean,
+  +primaryIndex?: 0 | 1,
+  +primaryMinSize?: number,
+  +secondaryMinSize?: number,
+  +secondaryInitialSize?: number,
+  +onDragStart?: () => mixed,
+  +onDragEnd?: () => mixed,
+  +onSecondaryPaneSizeChange?: number => mixed,
+  +children:
+    | $$reactsplitterlayout$$ReactElement
+    | [$$reactsplitterlayout$$ReactElement, $$reactsplitterlayout$$ReactElement],
```

Comment:
The component accepts more than 2 elements, but renders only the first 2, so I thought it's better to force it in the flow definition. What do you think?